google/resource_container_cluster.go

```diff
@@ -279,6 +279,48 @@ func resourceContainerCluster() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"ip_allocation_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
Any reason why this is (and the others are) computed?

Reply:
🤦‍♂️